map_model/src/make/parking_lots.rs

```diff
@@ -146,9 +149,24 @@ pub fn make_all_parking_lots(
     }
 
     timer.start_iter("generate parking lot spots", results.len());
-    for lot in results.iter_mut() {
+    for (lot, osm_tags) in results.iter_mut().zip(osm_tags.into_iter()) {
         timer.next();
         lot.spots = infer_spots(&lot.polygon, &lot.aisles);
+
+        // Guess how many extra spots are available, that maybe aren't renderable.
```

Comment:
There are other strategies here, like maybe using [capacity](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dparking#Additional_Tags), but this seems like a reasonable step